server/fleet/agent_options.go

```diff
@@ -14,6 +14,8 @@ type AgentOptions struct {
 	Overrides AgentOptionsOverrides `json:"overrides,omitempty"`
 	// CommandLineStartUpFlags are the osquery CLI_FLAGS
 	CommandLineStartUpFlags json.RawMessage `json:"command_line_flags,omitempty"`
+	// Extensions are the orbit managed extensions
+	Extensions json.RawMessage `json:"extensions,omitempty"`
```

Comment:
Maybe this should use the [ExtensionInfo](https://github.com/fleetdm/fleet/pull/8906/files#diff-bc31589a4fd9aed0a59f1bd77a489e2f80edc722517d3290191665bca50e33d1R228-R232) type, and then that type can be imported into the runner? If that is possible, it would allow more accurate validation.

Speaking of that, does Fleet automatically validate this with the type added here? Do we need to do anything else to support the validation?

Reply:
okay, let me chew on this, and see if we can use `ExtensionInfo` type, better validation is always nice! I followed a similar idiom as with the flags update. 

>Speaking of that, does Fleet automatically validate this with the type added here? Do we need to do anything else to support the validation?

Yep, Fleet does do a basic validation of `extensions`

Reply:
+1 to using `ExtensionInfo` instead of a `json.RawMessage`.

Reply:
Yep! Will be addressing it in #9046